etc/openstack-config/openstack-config.yml

```diff
@@ -1,60 +1,192 @@
 ---
 ###############################################################################
-# Configuration of OpenStack projects and users user environment.
+# Configuration of nova flavors for openstack.
 
-# List of OpenStack projects. Format is as required by the stackhpc.os-projects
-# role.
-#openstack_projects:
+# List of nova flavors in the openstack demo project. Format is as required by the
+# stackhpc.os-flavors role.
+openstack_flavors:
+  - "{{ openstack_flavor_m1_tiny }}"
+  - "{{ openstack_flavor_m1_small }}"
+  - "{{ openstack_flavor_m1_medium }}"
+  - "{{ openstack_flavor_m1_large }}"
+  - "{{ openstack_flavor_m1_xlarge }}"
+
+# Virtual machine flavors.
+openstack_flavor_m1_tiny:
+  name: "m1.tiny"
+  ram: 512
+  disk: 1
+  vcpus: 1
+
+openstack_flavor_m1_small:
+  name: "m1.small"
+  ram: 2048
+  disk: 20
+  vcpus: 1
+
+openstack_flavor_m1_medium:
+  name: "m1.medium"
+  ram: 4096
+  disk: 40
+  vcpus: 2
+
+openstack_flavor_m1_large:
+  name: "m1.large"
+  ram: 8192
+  disk: 40
+  vcpus: 4
+
+openstack_flavor_m1_xlarge:
+  name: "m1.xlarge"
+  ram: 16384
+  disk: 100
+  vcpus: 8
+
+openstack_images:
+  - "{{ openstack_image_cirros_0_6_0 }}"
+
+openstack_image_cirros_0_6_0:
+  name: "cirros"
+  type: qcow2
+  image_url: "https://github.com/cirros-dev/cirros/releases/download/0.6.0/cirros-0.6.0-x86_64-disk.img"
+  is_public: True
+  properties:
+    os_type: "linux"
+    os_distro: "cirros"
+    os_version: "0.6.0"
+    hw_rng_model: "virtio"
 
 ###############################################################################
-# Configuration of networks, subnets and routers.
+# Configuration of networks, subnets and routers for openstack.
 
 # List of networks in the openstack system. Format is as required by the
 # stackhpc.os-networks role.
-#openstack_networks:
-
-# List of routers in the openstack project. Format is as required by the
+openstack_networks:
+  - "{{ openstack_network_external }}"
+  - "{{ openstack_network_admin_vxlan }}"
+  - "{{ openstack_network_admin_vlan }}"
+  - "{{ openstack_network_admin_provider }}"
+
+# openstack external network name.
+openstack_network_external_name: "external"
+
+# openstack external network.
+openstack_network_external:
+  name: "{{ openstack_network_external_name }}"
+  project: "admin"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 102
+  shared: true
+  external: true
+  # Subnet configuration.
+  subnets:
+    - "{{ openstack_subnet_external }}"
+
+# openstack external subnet.
+openstack_subnet_external:
+  name: "{{ openstack_network_external_name }}"
+  project: "admin"
+  cidr: "192.168.38.0/24"
+  no_gateway_ip: false
```

Comment:
I think it should be true

Reply:
Changing this to `true` allows three tests to pass.

Also the `tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_connectivity_between_vms_on_different_networks` error changes to 

`File "/home/rally/.rally/verification/verifier-51551059-802e-4262-93f8-80a54db4fbf5/.venv/lib/python3.8/site-packages/netaddr/ip/__init__.py", line 305, in __init__
    raise AddrFormatError('failed to detect a valid IP ' \
netaddr.core.AddrFormatError: failed to detect a valid IP address from None`